### ISSUE TITLE
The pluign falsely detects invalid links (the links work)

### ISSUE THREAD

**Issue by mProjectsCode** [HiDeoo/starlight-links-validator#23]
### Describe the bug

For [my docs page](https://github.com/mProjectsCode/obsidian-meta-bind-plugin-docs), the link validation fails, even though all links are functional.

I am using the following base URL `/obsidian-meta-bind-plugin-docs`.
Links look like this `[link](/obsidian-meta-bind-plugin-docs/guides/...)`
Those are the error I am getting:
![image](https://github.com/HiDeoo/starlight-links-validator/assets/61602848/26cbefec-2a83-4c23-bd73-29d9c6d2e9fd)

### To Reproduce

I am not quite sure why this happens, but I believe it has to do with the base URL.

### Expected behavior

The link checker does not detect any broken links.

### How often does this bug happen?

Every time

### System Info

```json
"@astrojs/starlight": "^0.15.0",
"astro": "^4.0.3",
"starlight-links-validator": "^0.5.0"
```

**Comment by HiDeoo** [comment]
Thanks for your report :raised_hands: 

This should now be fixed in [`v0.5.1`](https://github.com/HiDeoo/starlight-links-validator/releases/tag/v0.5.1).

Before publishing the new version, I tested locally on your documentation website, and indeed, you were right, the links were valid as expected:

<img width="289" alt="SCR-20231214-qnda" src="https://github.com/HiDeoo/starlight-links-validator/assets/494699/c4822093-3f04-43fb-8ca2-ca6ecc7757ce">

**Comment by JuanM04** [comment]
I think this still happens for static files. For example, the file `public/document.pdf` with a `base: '/obsidian-meta-bind-plugin-docs/'` would get the link `/obsidian-meta-bind-plugin-docs/document.pdf`.

**Comment by HiDeoo** [comment]
> I think this still happens for static files. For example, the file `public/document.pdf` with a `base: '/obsidian-meta-bind-plugin-docs/'` would get the link `/obsidian-meta-bind-plugin-docs/document.pdf`.

Thanks for letting me know @JuanM04. This should now be fixed in [v0.5.2](https://github.com/HiDeoo/starlight-links-validator/releases/tag/v0.5.2).
